src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write code here
+
+const body = document.querySelector('tbody');
+const rows = body.querySelectorAll('tr');
+const title = document.querySelector('thead');
+
+title.addEventListener('click', (e) => {
+  const index = e.target.cellIndex;
+  const sorted = Array.from(rows).sort((A, B) => {
```

Comment:
don't use the variable or parameters in LowerCase, you need to use CamelCame for that